app/endpoints.py

```diff
@@ -0,0 +1,15 @@
+"""End point definition."""
```

Comment:
Could have this just before where you are now making up the Resources to make it more meaningful